CHANGELOG.md

```diff
@@ -38,6 +38,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Remove `_start_time_unix_nano` attribute from `_ViewInstrumentMatch` in favor
   of using `time_ns()` at the moment when the aggregation object is created
   ([#4137](https://github.com/open-telemetry/opentelemetry-python/pull/4137))
+- Include metric info in encoding exceptions
```

Comment:
This must be moved to unreleased

Reply:
Thanks -- good catch. I wonder if it got moved when I accepted a change earlier in this PR because it was in unreleased on my local branch. Anyway, should be fixed now.